syntax = "proto3";

option go_package = "github.com/npc_ric/pkg/grpc/table";

package table;

service TableService {
  rpc GetTable (GetTableRequest) returns (GetTableResponse);
}

message GetTableRequest {
  int32 institution_id = 1;
}

message GetTableResponse {
  repeated string columns = 1;
  repeated string rows = 2;
}